config_defaults/control.ini

```diff
@@ -78,6 +78,10 @@ key_match = docker-autotest
 product = Red Hat Enterprise Linux 7
 component = docker, docker-distribution, docker-registry, atomic,
             rhel-server-atomic, rhel-server-docker, rhel-tools-docker,
+<<<<<<< Updated upstream
```

Comment:
No, you need to resolve the conflict. That's waaaay beyond the scope of what I can help with in a github message. Can you read up on git, and git merges in particular, and/or ask someone local for help?

Merge conflicts are complicated and very, very important to understand. Please feel free to take a day or two to read, including working with examples in a scratch repo, and develop a basic understanding.